lindeck-frontend/src/common/classes/LayoutObject.jsx

```diff
@@ -0,0 +1,16 @@
+// Grid layout -> https://github.com/STRML/react-grid-layout
+export default class LayoutObject {
+    i = "0"
```

Comment:
Please think about variable names

Reply:
hrome-error://chromewebdata
not my false :(